src/main/resources/db/migration/V003__Add_is_failed_column.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE letters
+ADD COLUMN is_failed BOOLEAN;
```

Comment:
Would it make sense to update `LetterStatus` class with this field?

Reply:
Yes ofcourse @KarolJastrzebski Will do that as part of separate PR.Hope that is fine.

Reply:
i vote for separate pr. once this one is cleared up 👍